x-pack/solutions/security/plugins/lists/server/services/exception_lists/build_exception_filter.ts

```diff
@@ -329,6 +340,16 @@ export const buildExceptionFilter = async <
     listClient,
   });
 
+  agent.setCustomContext({
+    exceptions: {
```

Comment:
Related to the above comment: am I reading correct that this context has a different schema (an outer `exceptions` key) than the "zero'd" context above? Is that intentional?

Reply:
That was a mistake, thanks 👍